### ISSUE TITLE
Cookie Control: Make sure hidden elements are not focusable

### ISSUE THREAD

**Issue by MangopearUK** [MangopearUK/European-Boating-Association--Theme#392]

**Comment by MangopearUK** [comment]
Cookie Control removed from site
